tests/test_dagmc_query.py

```diff
@@ -182,3 +181,109 @@ def test_calc_tris_per_vert_dimension_incorrect():
             three_vols.root_set, types.MBVERTEX))
         test_pass[2] = (len(three_vols_query._vert_data['tri_per_vert']) == exp_tpv_len)
     assert(all(test_pass))
+
+def test_calc_area_triangle_vol():
+    """Tests part of the calc_area_triangle function
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    vol = three_vols.entityset_ranges['volumes'][0]
+    three_vols_query = dq.DagmcQuery(three_vols, vol)
+    three_vols_query.calc_area_triangle()
+    assert(all(three_vols_query._tri_data['area']) == all(np.full(12, 50)))
```

Comment:
12 and 50 are magic numbers in this line. Can you add variables to explain what they are? I think 12 is the number of triangles and 50 is the expected aspect ratio?

Reply:
(This comment applies to all the tests where such numbers appear)